core/modules/node/node.install

```diff
@@ -2108,6 +2108,789 @@ function node_update_1022() {
+            'summary' => '',
+            'columns' => 
+            array (
+              'title' => 'title',
+              'timestamp' => 'title',
+              'name' => 'title',
+              'edit_node' => 'edit_node',
+              'delete_node' => 'delete_node',
+            ),
+            'info' => 
+            array (
+              'title' => 
+              array (
+                'sortable' => 0,
+                'default_sort_order' => 'asc',
+                'align' => 'views-align-left',
+                'separator' => '',
+                'empty_column' => 0,
+              ),
+              'timestamp' => 
+              array (
+                'align' => '',
+                'separator' => '',
+                'empty_column' => 0,
+              ),
+              'name' => 
+              array (
+                'sortable' => 0,
+                'default_sort_order' => 'asc',
+                'align' => '',
+                'separator' => '',
+                'empty_column' => 0,
+              ),
+              'edit_node' => 
+              array (
+                'align' => '',
+                'separator' => '',
+                'empty_column' => 1,
+              ),
+              'delete_node' => 
+              array (
+                'align' => '',
+                'separator' => '',
+                'empty_column' => 1,
+              ),
+            ),
+            'default' => '-1',
+            'empty_table' => 0,
+          ),
+        ),
+      ),
+      'block' => 
+      array (
+        'display_title' => 'Block',
+        'display_plugin' => 'block',
+        'display_options' => 
+        array (
+          'query' => 
+          array (
+            'type' => 'views_query',
+            'options' => 
+            array (
+            ),
+          ),
+          'allow' => 
+          array (
+            'items_per_page' => 'items_per_page',
+            'exposed_form' => 'exposed_form',
+            'use_pager' => 0,
+            'offset' => 0,
+            'link_to_view' => 0,
+            'more_link' => 0,
+            'path_override' => 0,
+            'fields_override' => 0,
+          ),
+        ),
+      ),
+    ),
+  );
+  $config = config('views.view.recent_content');
+  if ($config->isNew()) {
+    $config->setData($data);
+    $config->save();
+  }
+
+  // Upgrade recent content block to views
+  $layouts= config_get_names_with_prefix('layout');
+  foreach ($layouts as $layout_name) {
+    $config = config($layout_name);
+    $configs = $config->get();
+    if (isset($configs['content'])) {
+      foreach ($configs['content'] as $uuid => $block) {
+        if ($block['data']['delta'] == 'recent' && $block['data']['module'] == 'node') {
+          $config->set('content.' . $uuid . '.data.delta', 'recent_content-block');
+          $config->set('content.' . $uuid . '.data.module', 'views');
+          $config->set('content.' . $uuid . '.plugin', 'views:recent_content-block');
```

Comment:
It would also help to add an additional class here for `block-node-recent` to make the previous selector line up with the new one.

![image](https://github.com/user-attachments/assets/1f547e2c-4d41-4309-bedd-d6fb0fa898d4)